package.json

```diff
@@ -66,5 +66,6 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
-  }
+  },
+  "packageManager": "pnpm@9.12.1+sha512.e5a7e52a4183a02d5931057f7a0dbff9d5e9ce3161e33fa68ae392125b79282a8a8a470a51dfc8a0ed86221442eb2fb57019b0990ed24fab519bf0e1bc5ccfc4"
```

Comment:
@imprashast it may not be relevant anymore but I remember us having issues with CI or something when we specified a version in `packageManager` - we removed that e.g. from @warp-ds/css in https://github.com/warp-ds/css/pull/208

Reply:
Let's see 🤞